.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: python
+env: # These should match the tox env list
+    - TOXENV=py27
+install: pip install coveralls tox
+script: tox
+# Special snowflake.  Our tests depend on a credentials_secret.json
+before_install:
+    - cp tests/json/credentials.json tests/json/credentials_secret.json
```

Comment:
‎Was thinking of setting the credentials file name in env so Travis uses the dummy file but this works too!                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         From: Anthony SottileSent: Thursday, November 12, 2015 8:26 AMTo: Yelp/yelp-pythonReply To: Yelp/yelp-pythonCc: Dinah ShiSubject: Re: [yelp-python] Makes the library more library-like (#23)In .travis.yml:
> @@ -0,0 +1,15 @@
> +language: python
> +env: # These should match the tox env list
> +    - TOXENV=py27
> +install: pip install coveralls tox
> +script: tox
> +# Special snowflake.  Our tests depend on a credentials_secret.json
> +before_install:
> +    - cp tests/json/credentials.json tests/json/credentials_secret.json

seems to work as long as the vcr casettes are there.

—Reply to this email directly or view it on GitHub.